c2rust-refactor/src/scripting/utils.rs

```diff
@@ -0,0 +1,16 @@
+use rlua::prelude::{LuaContext, LuaResult, LuaTable, ToLua};
+
+pub(crate) fn iter_to_lua_array<'lua, I, T: 'lua>(iter: I, lua_ctx: LuaContext<'lua>) -> LuaResult<LuaTable<'lua>>
```

Comment:
I wonder if it would be more ergonomic to make this into a mini-trait with a method `into_lua_array` implemented for any `Iterator`.

Reply:
It's an interesting idea, but with a trait you still have to import it and call it which is pretty similar to this function, no? I guess it just doesn't seem any more or less ergonomic?

Reply:
Yeah a trait would require an additional import.